test/api/controllers/requestParsers/validators/validations/DateRangeValidationV2Spec.scala

```diff
@@ -33,15 +33,22 @@ class DateRangeValidationV2Spec extends UnitSpec {
         DateRangeValidationV2.validate(None, None) shouldBe List()
       }
     }
-    "return a list containing an error" when {
+
+    "return a list containing a RangeToDateBeforeFromDateError" when {
```

Comment:
Not officially part of the change I realise but this should also check the boundary conditions:
- it's ok when to and from represent the same date
- it's an error when to date represents the day before the from date